File: python/add_two_nums.py
l1 = [2,4,3]
l2 = [5,6,4]

def add_two_nums(l1, l2):
    mod1 = int(''.join(map(str, l1[::-1])))
    mod2 = int(''.join(map(str, l2[::-1])))
    total = mod1 + mod2
    back = [int(x) for x in str(total)][::-1]
    return back
    

print(add_two_nums(l1, l2))

File: javascript/linkedList/addTwoNums.js
// Input: l1 = [2,4,3], l2 = [5,6,4]
// Output: [7,0,8]
// Explanation: 342 + 465 = 807.
const l1 = [2, 4, 3];
const l2 = [5, 6, 4];

const addTwo = (l1, l2) => {
  const mod1 = Number(l1.reverse().join(""));
  const mod2 = Number(l2.reverse().join(""));
  const total = mod1 + mod2;
  const back = total
    .toString()
    .split("")
    .map((x) => Number(x));
  return back.reverse();
};

console.log(addTwo(l1, l2));
